src/pages/examples/vrap.mdx

```diff
@@ -0,0 +1,360 @@
+
+Every folder in the repository is a separate ``pinecard``.
+Since different programs take different inputs, the content of the folder might change from program to program.
+
+In the generic `metadata.txt` file we include any relevant information about the observable:
+```txt
+description='E605 Fixed-Target Drell-Yan cross section measurement with nuclear copper target'
+hepdata=10.17182/hepdata.22831/t1-t8
+data_url=
+nnpdf_id=DYE605
+fktable_id=DYE605
+x1_label=sqrt(tau)
+x1_label_tex=$\sqrt(\tau)$
+x1_unit=
+x2_label=y
+x2_label_tex=$y$
+x2_unit=
+y_label_tex = $s*\frac{d^2\sigma}{d\sqrt(\tau)dy}$
+y_label=s*d^2sigma/dsqrt(tau)/dy
+y_unit=pb GeV^2
+```
+
+Then the information necessary to run the program is given as a human (and computer) readable `.yaml` file.
+
+```yaml
+Collider: piso
+E_CM: 38.8
+Alphat: 0.00757002271
+Q: 0
+VectorBoson: Zgamma
+muFoverQ: 1.0
+muRoverQ: 1.0
+Nf: 5
+UseOtherPDF: "yes"
+PDFset: 0
+AlphasZ: 0.118
+Order: NLO
+PrintDirection: Forward
+NNLO_only: "no"
+RandomSeed: 22334
+```
+
+Note that some of the information on this file, such as `Order` will be rewritten by the translation layer that we have prepared in the previous steps.
+Although in this example we have limited ourselves to the order of the calculation, in principle all theory-related information should be appropriately modified by the translation layer.
+
+Finally, in the case of `hawaiian vrap` the kinematics are provided as a separate `.dat` file containing the values of Q and rapidity for the desired points.
+```
+7.10428  -0.2
+7.30604  -0.2
+7.5078  -0.2
+```
+
+Now we are ready to run:
+```bash
+pinefarm run pinecards/DYE605 theories/theory_200_1.yaml
+````
+the results will be output to the `results/DYE605-<timestamp>` folder.
+
+## Evolution kernel operators and FKTables
+
+In the output folder of the run one can find the runcard that have been used for the run of the Monte Carlo alongside the output results and the final `pineappl` grid.
+
+This grid is differential in $x_{1}$, $x_{2}$ and $Q^{2}$.
+The PDF fit is however done at a fixed scale $Q_{0}^{2}$.
+In order to avoid having to evolve the PDF to the process scale 
+we are going to construct Evolution Kernel Operator using EKO[^4] to instead evolve the grid down to the fitting scale $Q_{0}^{2}$.
+
+In the NNPDF language the grids convoluted with an EKO are called Fast-Kernel (FK) Tables.
+The program that takes `pineappl`(s) and makes it into `FKTables` is called `pineko`.
+
+The usage of `pineko` is straightforward thanks to the bootstrap command including with the software.
+There are detailed guides for the [set-up](../tutorials/setup) and [generation of FK Tables](../tutorials/fktable).
+
+The following steps are those of the [FK Table](../tutorials/fktable)
+so we will outline here the specific differences for our dataset.
+
+We need a small file defining any strange operations to be done with the dataset.
```

Comment:
I believe in the `pineko` tutorial I have briefly described what kind of `yaml` file you need. Maybe you can cite that part (just for completeness)

Reply:
It's linked in the line above ([Fk Table](../tutorials/fktable)) 

or do you mean linking specifically the yaml part? (I guess adding `#1-yaml-file` will work?)

Reply:
Yes I mean the yaml part because I believe it is a bit confusing that you say "The following steps are those of the [FK Table](../tutorials/fktable) so we will outline here the specific differences for our dataset" but then you explain the yaml file that has also been explained there (and basically in the same way).

Reply:
Ah, no, it's just a sentence to explain why the one for this dataset is different from the one in that tutorial. But ok I'll clarify!

Reply:
Ok thank you, It was not extremely clear for me :)